modules/tensor_mechanics/src/materials/ComputeLinearElasticPFFractureStress.C

```diff
@@ -110,7 +111,8 @@ ComputeLinearElasticPFFractureStress::computeStrainSpectral(Real & F_pos, Real &
     _d2Fdcdstrain[_qp] = stress0pos * _dDdc[_qp];
 
   // Used in StressDivergencePFFracTensors off-diagonal Jacobian
-  _dstress_dc[_qp] = stress0pos * _dDdc[_qp];
+  _dstress_dc[_qp] = stress0pos * _dDdc[_qp] +
+                     _pressure[_qp] * RankTwoTensor(RankTwoTensor::initIdentity) * _dIdc[_qp];
```

Comment:
Something's not right either in _dstress_dc or in _stress. There is a sign mismatch.